simul/platform/cliutils.go

```diff
@@ -23,14 +27,38 @@ func Scp(username, host, file, dest string) error {
 	return cmd.Run()
 }
 
+// Copy makes a copy of a local file with the same file-mode-bits set.
+func Copy(dst, src string) error {
```

Comment:
Wait this is already in app/lib/<something>, so if we need it at least twice, then we should have a `fs` package or `utils` package at the top level where we could put such functions.